internal/driver/glfw/window_desktop.go

```diff
@@ -677,6 +678,32 @@ func (w *window) charInput(viewport *glfw.Window, char rune) {
 	w.processCharInput(char)
 }
 
+var counterIme int
+
+// IME
+func (w *window) imeStatus(_ *glfw.Window) {
+	fmt.Println("ime status ", counterIme)
+	counterIme++
+}
+func (w *window) preedit(
+	_ *glfw.Window,
+	preeditCount int,
+	preeditString string,
+	blockCount int,
+	blockSizes string,
+	focusedBlock int,
+	caret int,
+) {
+	fmt.Println(
+		"preedit preeditCount", preeditCount,
+		"preeditString", preeditString,
+		"blockCount", blockCount,
+		"focusedBlock", focusedBlock,
+		"caret", caret,
+	)
+	w.processPreedit(preeditString)
```

Comment:
It looks like it would be easier to just inline processPreedit here as it is barely a single line and only called from this location

Reply:
@andydotxyz  Unfortunately processPreedit and Preeditable.PreeditChanged are temporary interfaces. This is because the original Preedit response requires a more complex response. It is necessary to additionally display the conversion candidates for input content provided by the IME and the IME's own caret.

Reply:
OK. Should we mark this as a draft PR then instead of one ready for review?